packages/server/__snapshots__/run_plugins_spec.coffee

```diff
@@ -1,21 +1,65 @@
-exports['lib/plugins/child/run_plugins sends error message if pluginsFile is missing 1'] = {
-  "name": "Error",
-  "message": "Cannot find module '/does/not/exist.coffee'",
-  "code": "MODULE_NOT_FOUND"
-}
+exports['lib/plugins/child/run_plugins sends error message if pluginsFile is missing 1'] = `Error: Cannot find module '/does/not/exist.coffee'
+    at Function.Module._resolveFilename (<path>)
+    at Module._load (<path>)
+    at Function.hookedLoader [as _load] (<path>)
+    at Module.require (<path>)
+    at require (<path>)
+    at module.exports.err (<path>)
+    at Context.<anonymous> (<path>)
+    at callFn (<path>)
+    at Test.Runnable.run (<path>)
+    at Runner.runTest (<path>)
+    at(<path>)
+    at next (<path>)
+    at(<path>)
+    at next (<path>)
+    at(<path>)
+    at done (<path>)
+    at callFn (<path>)
+    at Hook.Runnable.run (<path>)
+    at next (<path>)
+    at Immediate.<anonymous> (<path>)
+    at runCallback (<path>)
+    at tryOnImmediate (<path>)
+    at processImmediate [as _immediateCallback] (<path>)
+`
 
-exports['lib/plugins/child/run_plugins sends error message if requiring pluginsFile errors 1'] = {
-  "name": "Error",
-  "message": "error thrown by pluginsFile"
-}
+exports['lib/plugins/child/run_plugins sends error message if requiring pluginsFile errors 1'] = `Error: error thrown by pluginsFile
+    at Object.<anonymous> (<path>)
+    at Object.<anonymous> (<path>)
+    at Module._compile (<path>)
+    at Object.loadFile (<path>)
+    at Module.load (<path>)
+    at tryModuleLoad (<path>)
+    at Module._load (<path>)
+    at Function.hookedLoader [as _load] (<path>)
+    at Module.require (<path>)
+    at require (<path>)
+    at module.exports.err (<path>)
+    at Context.<anonymous> (<path>)
+    at callFn (<path>)
+    at Test.Runnable.run (<path>)
+    at Runner.runTest (<path>)
+    at(<path>)
+    at next (<path>)
+    at(<path>)
+    at next (<path>)
+    at(<path>)
+    at done (<path>)
+    at callFn (<path>)
+    at Hook.Runnable.run (<path>)
+    at next (<path>)
+    at Immediate.<anonymous> (<path>)
+    at runCallback (<path>)
+    at tryOnImmediate (<path>)
+    at processImmediate [as _immediateCallback] (<path>)
+`
 
-exports['lib/plugins/child/run_plugins sends error message if pluginsFile has syntax error 1'] = {
-  "name": "SyntaxError",
-  "message": "missing }",
-  "code": "{\n"
-}
+exports['lib/plugins/child/run_plugins sends error message if pluginsFile has syntax error 1'] = `/Users/chrisbreiding/Dev/cypress/cypress/packages/server/test/fixtures/syntax_error.coffee:1:1: error: missing }
```

Comment:
hmm, this looks like something went wrong in this test snapshot

Reply:
How so?

Reply:
I see this in the diff lines, which is confusing - maybe we should strip colors from the text output

<img width="273" alt="screen shot 2017-11-21 at 1 51 20 pm" src="https://user-images.githubusercontent.com/2212006/33090750-1dcd134a-cec3-11e7-91c5-cdd388972742.png">